golang/x/relay/client/cli/query.go

```diff
@@ -7,9 +7,18 @@ import (
 	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/spf13/cobra"
+	"github.com/summa-tx/bitcoin-spv/golang/btcspv"
 	"github.com/summa-tx/relays/golang/x/relay/types"
 )
 
+type QueryResGetParent struct {
+	Digest btcspv.Hash256Digest `json:"digest"`
```

Comment:
access as `types.Hash256Digest`

Reply:
may even be able to omit `types` here? :thinking:

Reply:
I missed this earlier, I'll include this in my next PR.  Looks like it has to be `types.`